app/views/merchant_items/index.html.erb

```diff
@@ -1,11 +1,26 @@
 <h1>Merchant Items:</h1>
 <hr>
-<% @merchant.items.each do |item| %>
 
-<p><b>Name:</b> <%= link_to "#{item.name}", "/merchants/#{@merchant.id}/items/#{item.id}" %> </p>
-<% if item.status == "Disabled" %>
-<%= button_to "Enable #{item.name}", "/items/#{item.id}?status=enable", method: :patch %>
-<% elsif item.status == "Enabled" %>
-<%= button_to "Disable #{item.name}", "/items/#{item.id}?status=disable", method: :patch %>
-<% end %>
-<% end %>
+<div class="enabled_items">
+  <ol>
+    <h2>Enabled Items:</h2>
+    <% @merchant.items.each do |item| %>
```

Comment:
Not going to make you do it on this one but next user story consider making this more readable